<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtHuffZip</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="BrowseButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="BrowseButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="3" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="FromFileRadioButton">
          <property name="text">
           <string>来自文件</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="FromTextRadioButton">
          <property name="text">
           <string>来自文本</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DeCompRadioButton">
          <property name="text">
           <string>解压模式</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="6" rowspan="8">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="DestSaveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save⬇️</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DestFetchButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Fetch⬆️</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="CompressButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Compress⬇️</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeCompressButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>DeCompress⬆️</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="OrigFetchButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Fetch⬇️</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="OrigSaveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save⬆️</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="6">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item row="0" column="7" rowspan="8">
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>打印字符</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>字节</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>哈夫曼编码</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>出现次数</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>占比</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>比例条(log)</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
